### PR TITLE
Infra styled components

### PR DESCRIPTION
**Pull request by yhaspel**
Converted components to Styled Components:
SmallLoader
SendReceiveButton
SmButton
SideMenu

Also moved the baseComponent directory one folder up.

### CODE REVIEW

File: app/baseComponents/SendReceiveButton/SendReceiveButton.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+// @flow
+/* eslint react/no-unused-prop-types: 0 */
```
Comment (IlyaVi) [inline app/baseComponents/SendReceiveButton/SendReceiveButton.js]: remove this comment

File: app/baseComponents/SideMenu/SideMenu.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        <StyledLoadingIconWrapper>
+          {loadingEntry && loadingEntry.id === entry.id && (
+            <SmallLoader isLoading={loadingEntry.isLoading} loadingLeft={isOpen ? 8 : -96} loadingSize={isOpen ? 18 : 25} loadingTop={isOpen ? 18 : 14} />
+          )}
+        </StyledLoadingIconWrapper>
+      </StyledMenuEntry>
+    );
+  };
+
+  renderTopOrBottomElem = (topOrBottom: 'top' | 'bottom') => {
```
Comment (IlyaVi) [inline app/baseComponents/SideMenu/SideMenu.js]: make this func same style as next one (immediate return)

File: app/baseComponents/SendReceiveButton/SendReceiveButton.js (hunk 1 above)
Comment (yhaspel) [reply app/baseComponents/SendReceiveButton/SendReceiveButton.js]: For some reason, in a functional component, eslint will always comment on "PropType defined but never used".
This is true for all required props... It's either this comment or one comment per prop.
What are your thoughts?

File: app/baseComponents/SideMenu/SideMenu.js (hunk 2 above)
Comment (yhaspel) [reply app/baseComponents/SideMenu/SideMenu.js]: done

**Pull request merged**